wiki/History_of_osu!/en.md

```diff
@@ -19,6 +19,11 @@ Records of osu!'s history, all in the osu! wiki.
 - [2016](2016) - scorev2, announcement of a new open-source "lazer" version of osu! (previously known as "osu!lazer"), and an osu! wiki overhaul
 - [2017](2017) - release of the first lazer build of osu!, and Discord integration on the stable client
 - [2018](2018) - ?
+- [2019](2019) - ?
+- [2020](2020) - ?
+- [2021](2021) - ?
+- [2022](2022) - 15 years of osu!, ?
+- [2023](2023) - ?
```

Comment:
if you are modelling this after 2018, consider that it at least has a dedicated page, while the years you have added links for don't, so it's better if you do it this way:
```suggestion
- 2019–2023 - ?
```

if you regret losing the 15 years remark, don't -- it's an event significant enough, so people will remember and write it down once they get to work on the 2022 article

Reply:
alternative option: `2019–present - ?`

Reply:
honestly, I agree with this changes, but 2018 has also a question mark

Reply:
see my comment below (it changes the 2018 line as well, as opposed to the suggestion in this comment thread)